### ISSUE TITLE
Error while building imagemagick

### ISSUE THREAD

**Issue by CityOfLight77** [zodf0055980/Yuan-fuzz#1]
I'm using gcc 9.3.0 pre-built binaries of clang+llvm 11 for ubuntu 20.04

```sh
clang version 11.0.0 (https://github.com/llvm/llvm-project.git 0160ad802e899c2922bc9b29564080c22eb0908c)
Target: x86_64-unknown-linux-gnu
Thread model: posix
InstalledDir: /usr/local/clang_11.0.0/bin
Found candidate GCC installation: /usr/lib/gcc/x86_64-linux-gnu/9
Selected GCC installation: /usr/lib/gcc/x86_64-linux-gnu/9
Candidate multilib: .;@m64
Selected multilib: .;@m64

/usr/local/clang_11.0.0/bin/clang
```
And here is command I used to build imagemagick

```
CC=~/Yuan-fuzz/afl-clang-fast CXX=~/Yuan-fuzz/afl-clang-fast++ CFLAGS="-g -O0 -fsanitize=address" CXXFLAGS="-g -O0 -fsanitize=address" ./configure
AFL_USE_ASAN=1 make -j4 && sudo make -j4 install
```

But I'm encountered error. Any idea why I got error?
```sh
/usr/bin/ld: /home/cityoflight/Yuan-fuzz/afl-llvm-rt.o:/home/cityoflight/Yuan-fuzz/llvm_mode/afl-llvm-rt.o.c:60: multiple definition of `__afl_area_initial'; /home/cityoflight/Yuan-fuzz/afl-llvm-rt.o:/home/cityoflight/Yuan-fuzz/llvm_mode/afl-llvm-rt.o.c:60: first defined here

/usr/bin/ld: /home/cityoflight/Yuan-fuzz/afl-llvm-rt.o:/home/cityoflight/Yuan-fuzz/llvm_mode/afl-llvm-rt.o.c:61: multiple definition of `__afl_area_ptr'; /home/cityoflight/Yuan-fuzz/afl-llvm-rt.o:/home/cityoflight/Yuan-fuzz/llvm_mode/afl-llvm-rt.o.c:61: first defined here

/usr/bin/ld: /home/cityoflight/Yuan-fuzz/afl-llvm-rt.o: in function `__afl_auto_init':
/home/cityoflight/Yuan-fuzz/llvm_mode/afl-llvm-rt.o.c:257: multiple definition of `__afl_auto_init'; /home/cityoflight/Yuan-fuzz/afl-llvm-rt.o:/home/cityoflight
/Yuan-fuzz/llvm_mode/afl-llvm-rt.o.c:257: first defined here

/usr/bin/ld: /home/cityoflight/Yuan-fuzz/afl-llvm-rt.o: in function `__afl_manual_init':
/home/cityoflight/Yuan-fuzz/llvm_mode/afl-llvm-rt.o.c:240: multiple definition of `__afl_manual_init'; /home/cityoflight/Yuan-fuzz/afl-llvm-rt.o:/home/cityoflight/Yuan-fuzz/llvm_mode/afl-llvm-rt.o.c:240: first defined here

/usr/bin/ld: /home/cityoflight/Yuan-fuzz/afl-llvm-rt.o: in function `__afl_persistent_loop':
/home/cityoflight/Yuan-fuzz/llvm_mode/afl-llvm-rt.o.c:184: multiple definition of `__afl_persistent_loop'; /home/cityoflight/Yuan-fuzz/afl-llvm-rt.o:/home/cityoflight/Yuan-fuzz/llvm_mode/afl-llvm-rt.o.c:184: first defined here

/usr/bin/ld: /home/cityoflight/Yuan-fuzz/afl-llvm-rt.o:(.tbss+0x0): multiple definition of `__afl_prev_loc'; /home/cityoflight/Yuan-fuzz/afl-llvm-rt.o:(.tbss+0x0): first defined here

/usr/bin/ld: /home/cityoflight/Yuan-fuzz/afl-llvm-rt.o: in function `__sanitizer_cov_trace_pc_guard':
/home/cityoflight/Yuan-fuzz/llvm_mode/afl-llvm-rt.o.c:276: multiple definition of `__sanitizer_cov_trace_pc_guard'; /home/cityoflight/Yuan-fuzz/afl-llvm-rt.o:/home/cityoflight/Yuan-fuzz/llvm_mode/afl-llvm-rt.o.c:276: first defined here

/usr/bin/ld: /home/cityoflight/Yuan-fuzz/afl-llvm-rt.o: in function `__sanitizer_cov_trace_pc_guard_init':
/home/cityoflight/Yuan-fuzz/llvm_mode/afl-llvm-rt.o.c:284: multiple definition of `__sanitizer_cov_trace_pc_guard_init'; /home/cityoflight/Yuan-fuzz/afl-llvm-rt.o:/h
ome/cityoflight/Yuan-fuzz/llvm_mode/afl-llvm-rt.o.c:284: first defined here
clang-11: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [Makefile:6065: Magick++/lib/libMagick++-6.Q16.la] Error 1
make[1]: Leaving directory '/home/cityoflight/ImageMagick-6.9.11-45'
make: *** [Makefile:5682: all] Error 2
```

**Comment by zodf0055980** [comment]
Hello, our compiler is the same as AFL.
And I try to compile using `afl-gcc` & `afl-g++` is success in `Ubuntu 18.04`

**Comment by CityOfLight77** [comment]
It's success now after appending `--disable-shared` flag to the configure.

Another problem is I got very slow execution, only 4 exec/s. Copying inputs and the target to `/run/shm` doesn't get any performance gain because what I saw from `df -h` the `/run/shm` directory is a temporary directory and already run tmpfs.

Here is the command:
`./Yuan-fuzz -i /run/shm/in -o /run/shm/out -m none -s <xml_path> -- /run/shm/convert`

Am I missing something?
